ProjectShop/ProductApp/models.py

```diff
@@ -0,0 +1,111 @@
+from django.db import models
+from django.core.validators import *
+from UserApp.models import User
+
+
+class ProductCategory(models.Model):
+
+    """
+    A database objects that represents a category of items.
+    (Like large groups: TVs, phones, tables, etc... SUBJECT TO CHANGE.)
+
+    Attributes:
+        name: name of the category.
+    """
+
+    name = models.CharField(max_length=100, null=False, blank=False)
+
```

Comment:
Add verbose name to all fields 
Add translations to all fields

from django.utils.translation import ugettext_lazy as _

name = models.CharField(verbose_name=_('name'), max_length=100, null=False, blank=False)